code/stable/dblpend/SRS/HTML/DblPend_SRS.html

```diff
@@ -866,15 +868,15 @@ <h4>
                     <p class="paragraph">
                       We also know the horizontal position that is defined in <a href=#DD:positionXDD1>DD:positionXDD1</a>
                     </p>
-                    \[p_{\text{x}1}=L_{1}\,\sin\left(θ_{1}\right)\]
+                    \[{p_{\text{x}1}}={L_{1}}\,\sin\left({θ_{1}}\right)\]
```

Comment:
Looks like the change added too many curlies, especially at the 'top level', as can be seen here.